### ISSUE TITLE
Agama storage "check the xy planned actions" link dead

### ISSUE THREAD

**Issue by shundhammer** [agama-project/agama#1749]
## Problem

Starting from the Agama storage tab:

![agama-storage-01-initial](https://github.com/user-attachments/assets/cfcf6f5b-bd07-4b5e-9536-4fbf3f48c415)

When I click on the "Actions" -> "Change" button to get here:

![agama-storage-07-change-actions](https://github.com/user-attachments/assets/6510d70d-426b-4b24-84a1-f082bd96b31a)

and I click the "Cancel" button to go back to the previous screen, the "Check the 32 planned actions" link is now dead: It does not show a popup with the actions anymore.

## Attempted Workarounds

When I change to another tab like "Overview" and back to "Storage", the link remains dead.

Also when trying to change the installation device and hit "Cancel" there as well; when I click "Accept" there, the link becomes active again and shows the actions in a popup.

Expanding "Partitions and file systems" and changing the "Use Btrfs snapshots" checkbox also enables it again.

## Used Image

Build 28.1 ISO from https://download.opensuse.org/repositories/systemsmanagement:/Agama:/Devel/images/iso/, downloaded on 2024-11-11.

**Comment by jreidinger** [comment]
yeah, forgot to report it, but it is also what happens to me during live demo on conference.

**Comment by shundhammer** [comment]
It's the same when trying to change the installation device and then closing that dialog with "Cancel".

**Comment by imobachgs** [comment]
This bug is obsolete because this UI has been replaced with something completely new. Thanks!
